agent/engine/docker_task_engine.go

```diff
@@ -1329,6 +1327,34 @@ func getFirelensLogConfig(task *apitask.Task, container *apicontainer.Container,
 	return logConfig
 }
 
+// getFirelensEnvironmentVariables returns env variables for app containers that use awsfirelens
+// for task using firelens v1, inject FLUENT_HOST to container
+// for task using firelens v2 FluentBit collector, inject FLUENT_HOST to container if not already configured by customer
+// for task using firelens v2 OpenTelemetry collector, inject OPEN_TELEMETRY_HOST to container if not already configured by customer
+func getFirelensEnvironmentVariables(task *apitask.Task, container *apicontainer.Container,
```

Comment:
could we add unit tests for this method?

Reply:
Added, thanks!!